roles/velero/defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+aws_secret_split: []
+aws_access_key_id: ''
+aws_secret_access_key: ''
```

Comment:
Not sure I would even bother putting these in defaults. We set them with `set_fact` so don't need a default.

Reply:
Done